plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/GodotPlugin.kt

```diff
@@ -35,19 +37,27 @@ class GodotPlugin : Plugin<Project> {
     private fun setupKotlinPlugin(project: Project, mpp: KotlinMultiplatformExtension, godot: GodotExtension) {
         project.afterEvaluate {
             // create the main source set
-            val godotMain = mpp.sourceSets.create(MAIN_SOURCE_SET_NAME)
-            val godotTest = mpp.sourceSets.create(TEST_SOURCE_SET_NAME)
+            // val godotMain = mpp.sourceSets.create(MAIN_SOURCE_SET_NAME)
+            // val godotTest = mpp.sourceSets.create(TEST_SOURCE_SET_NAME)
 
             fun KotlinNativeTarget.configureSourceSets(includeEntrySourceDir: Boolean) {
                 compilations.getByName("main").defaultSourceSet {
-                    dependsOn(godotMain)
+                    kotlin.srcDirs += project.file("src/$MAIN_SOURCE_SET_NAME/kotlin")
                     if (includeEntrySourceDir) {
                         kotlin.srcDirs += godot.entrySourceDir.get().asFile
                     }
+
+                    dependencies {
+                        // TODO: remove this once we have published the godot-library artifact.
```

Comment:
removing this will make IJ complain about not being able to resolve mac and windows versions of `godot-library` in linux. We can remove this once we publish the artifacts to jcenter or central.